var/spack/repos/builtin/packages/pipits/package.py

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Pipits(PythonPackage):
```

Comment:
Is there any reason we shouldn't name this `py-pipits` to match the other PythonPackages? There are only a few exceptions of PythonPackages that don't start with `py-`:

* busco
* easybuild
* httpie
* mercurial
* scons
* snakemake

My rule of thumb is that if the package uses the `PythonPackage` build system and installs Python libraries to `prefix/lib/python2.7/site-packages`, then it should be called `py-`, but it if is primarily intended to be a tool instead of an importable Python library (like mercurial and scons), then it shouldn't be named `py-`.